onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -148,9 +148,12 @@ def wrapped(fn):
 def duplicate_opinfo(opinfos: list[opinfo_core.OpInfo], name: str, new_names: tuple[str, ...]):
     """Duplicate an opinfo in the opinfo database and give it a new name."""
     duplicated = []
+    all_info_names = {opinfo.name for opinfo in opinfos}
     for opinfo in opinfos:
         if opinfo.name == name:
             for new_name in new_names:
+                if new_name in all_info_names:
+                    warnings.warn(f"OpInfo {new_name} already exists in the database.")
```

Comment:
NOTE: This is solving an issue caused by changes in latest upstream pytorch
```
ERROR onnxscript/test/function_libs/torch_aten/ops_correctness_test.py - AssertionError: Redefinition of test test_output_match_full_cpu_float32
```

Reply:
Added a note